Source/UltraleapTrackingCore/Private/LeapComponent.cpp

```diff
@@ -401,4 +401,36 @@ bool ULeapComponent::GetLeapOptions(FLeapOptions& Options)
 		}
 	}
 	return false;
+}
+
+void ULeapComponent::GetHandSize(float& OutHandSize)
```

Comment:
Might be worth commenting this method that we measure the middle finger length + palm position to start of finger as an indication of the hand size.